.azure/azure-pipelines.ci.yml

```diff
@@ -99,6 +99,24 @@ stages:
       tls: openssl
       config: Release
       extraBuildArgs: -EnableTelemetryAsserts
+  - template: ./templates/build-config-user.yml
```

Comment:
I just noticed we didn't enable any static builds for Linux. Should Linux work with this PR? If so, we should enable a build for it to make sure.

Reply:
Never mind. When reviewing the files below, I see Linux is not supported yet.